doc/nrf/ug_thingy53.rst

```diff
@@ -267,25 +239,30 @@ To build and program the source code from the command line, complete the followi
 Updating firmware image
 =======================
 
-The firmware on Thingy:53 can be programmed using an external debug probe and 10-pin JTAG cable, as described in the sections above for building in the different environments.
-You can also update applications running on both network and application core using the built-in MCUboot bootloader and `nRF Connect Programmer`_ or the nRF Programmer for Android or iOS.
-Prebuilt application images can also be updated that way.
-See subsections below for details about updating firmware image.
+You can program the firmware on the Nordic Thingy:53 using an external debug probe and 10-pin JTAG cable, as described in :ref:`thingy53_building_pgming`, using either :ref:`Visual Studio Code <thingy53_build_pgm_vscode>`, :ref:`SEGGER Embedded Studio <thingy53_build_pgm_segger>`, or :ref:` command line <thingy53_build_pgm_command_line>`.
+You can also update applications running on both the network and application core using the built-in MCUboot bootloader and `nRF Connect Programmer`_ or the `nRF Programmer`_ app for Android and iOS.
+You can also update the prebuilt application images that way.
+
+See :ref:`thingy53_gs_updating_firmware` for details about updating firmware image.
+
+.. _thingy53_app_update_debug:
 
 Firmware update using external debug probe
 ------------------------------------------
 
 If you are using an external debug probe, such as the nRF5340 DK, or any J-Link device supporting ARM Cortex-M33, you do not have to use applications that follow the :ref:`thingy53_app_guide`.
-In that case, you can program the Thingy:53 in a similar way as nRF53 DK.
+In that case, you can program the Thingy:53 in a similar way as nRF5340 DK.
 See :ref:`ug_nrf5340` for details.
-Keep in mind that you will need to program a sample or application compatible with Thingy:53 application guide to bring back serial recovery and DFU OTA.
+
+.. note::
+   You need to program a sample or application compatible with the Nordic :ref:`thingy53_app_guide` to bring back serial recovery and DFU OTA.
 
 .. _thingy53_app_update_mcuboot:
 
 Firmware update using MCUboot bootloader
 ----------------------------------------
 
-The Thingy:53-compatible samples and applications use MCUboot bootloader than can be used to update firmware.
+The Thingy:53-compatible samples and applications use MCUboot bootloader that you can use to update firmware.
```

Comment:
Suggest revising to: The Thingy:53-compatible samples and applications include a MCUboot bootloader that you can use to update firmware.

Reply:
Applied.